gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageConfiguration.java

```diff
@@ -103,6 +103,7 @@ public Builder blockSize(int value) {
       return this;
     }
 
+
     /** Creates a new instance, but does not destroy the builder. */
     public CloudStorageConfiguration build() {
```

Comment:
Indeed this one without any At-clause (param, throws,...) is fine by the public [Google Checkstyle](http://checkstyle.sourceforge.net/google_style.html) however in the other cases it isn't.

For this reason and for consistency with the rest of `gcloud-java` it would be better to avoid this special case.